float64 start_x
float64 start_y
float64 end_x
float64 end_y
float64 radius
uint32 obstacle_num
---
bool is_path_planned
float64 calculation_time_ms
